docs/reference/style/pack.rst

```diff
@@ -287,11 +287,16 @@ The weight of the font to be used.
 ``font_size``
 -------------
 
-**Values:** ``<integer>``
+**Values:**
+    - ``<integer>`` (in :ref:`CSS points <css-units>`)
+    - Absolute keywords: ``xx-small`` | ``x-small`` | ``small`` | ``medium`` | ``large`` | ``x-large`` | ``xx-large``
 
 **Initial value:** System default
 
-The size of the font to be used, in :ref:`CSS points <css-units>`.
+The size of the font to be used. Can be specified in three ways:
```

Comment:
Better not to include a number: it's easy for it to become incorrect if the list is edited in the future.

```suggestion
The size of the font to be used. Can be specified in the foillowing ways:
```